spitexorganizer/src/main/java/ch/bfh/bti7081/s2019/black/spitexorganizer/DataInit.java

```diff
@@ -26,28 +28,33 @@
 @Component
 public class DataInit implements ApplicationRunner {
 
-    @Autowired
-    private AppointmentRepository appointmentRepository;
-
-    @Autowired
-    private PatientRepository patientRepository;
-
-    @Autowired
-    private ReportRepository reportRepository;
-
-    @Autowired
-    private EmployeeRepository employeeRepository;
-
-    @Autowired
-    private TaskRepository taskRepository;
+    private final AppointmentRepository appointmentRepository;
+    private final PatientRepository patientRepository;
+    private final ReportRepository reportRepository;
+    private final EmployeeRepository employeeRepository;
+    private final TaskRepository taskRepository;
+    private final EvaluationRepository evaluationRepository;
+    private final PasswordEncoder passwordEncoder;
 
     @Autowired
-    private EvaluationRepository evaluationRepository;
+    public DataInit(AppointmentRepository appointmentRepository, PatientRepository patientRepository, ReportRepository reportRepository, EmployeeRepository employeeRepository, TaskRepository taskRepository, EvaluationRepository evaluationRepository, PasswordEncoder passwordEncoder) {
+        this.appointmentRepository = appointmentRepository;
+        this.patientRepository = patientRepository;
+        this.reportRepository = reportRepository;
+        this.employeeRepository = employeeRepository;
+        this.taskRepository = taskRepository;
+        this.evaluationRepository = evaluationRepository;
+        this.passwordEncoder = passwordEncoder;
+    }
 
```

Comment:
Wise autowired geändert?

Reply:
Intellij zeigt mir "Field injection not recommended" und anstatt den Fehler zu ignorieren habe ich Alt+Enter -> Fix geklickt.